app/gulpfile.js

```diff
@@ -19,29 +23,36 @@ gulp.task('buildMap', function(){
     browserifyShare('Map');
 });
 
-gulp.task('watch', function() {
+gulp.task('watch-app', function() {
```

Comment:
Per discussion:
Ok for the first point.
No need for the second point as the 2 gulp tasks will run in different dockers so the behavior has to be duplicated.